www/css/app.css

```diff
@@ -49,6 +49,11 @@
   padding: 0;
 }
 
+.panel-body p {
+    text-indent: -22px;
+    margin-left: 20px;
+}
```

Comment:
What is the purpose of this style overriding?
I don't see a big difference if I remove it (at least on Firefox desktop)

Reply:
Alignment of the text after radio buttons (or checkbox), so that, if it wraps (on narrow screens) it aligns with the start of the first word, and not with the extreme left of the radio button, which looks ugly...

Reply:
It seems to be possible to do that while keeping the <label> tag (I find it important, as it allows the user to click on the label to select it, instead of only the radio-button itself), and avoiding the fixed pixel numbers. With display:table-* : I've put an example on http://jsfiddle.net/z59bkdo0/2/
This seems to be widely supported by browsers : http://caniuse.com/#feat=css-table
NB : I'm not a CSS expert, so not sure it's the best way

Reply:
Great, I wasn't aware of this (and t.b.h. originally when I changed the labels to paras it was a "quick fix" I'd since forgotten about).  Will experiment.